.gitattributes

```diff
@@ -1,39 +1,64 @@
+# Git LFS inclusion / exclusion is controlled by this file. Our approach is to:
+# INCLUDE dirs in Git LFS broadly, and then EXCLUDE text file-types narrowly.
+#
+# Why whitelist? Once a large binary file is committed to LFS, it is VERY
```

Comment:
nit: Prefer use of an alternative to whitelist/blacklist. Maybe "allow list"?

Reply:
Also, something about these two sentences I need clarification on. I understand broadly that your approach prioritizes managing repository size and complexity while not worrying about the ability to correct errors with narrowly excluded smaller files. But your comment says, "Why whitelist?" which, upon first read, I think refers to "including in LFS", and then you describe how accidental inclusion in LFS is really hard to undo. So I'm reading:
```
why be inclusive of large files? --> because accidentally including large files is a really tough mistake to undo
```
But maybe how I'm meant to read this is,
```
why narrowly exclude? --> unlike big files, undoing a mistake with a small file is easy
```

I think there's something about the "whitelist" meant to stand for "exclusion" that is making this hard for me to read, and maybe I'm the only person who would get hung up on this, but since I had to sit and think about it for a few minutes, I figured it's worth mentioning 🤷‍♀️

Reply:
Backwards.... whitelist here was intended to refer to "narrow inclusion in LFS". I'll just use include/exclude.

Its not that small vs large file mistakes are easy to correct. Its that INCLUDING IN LFS is very hard to undo, but EXCLUDING FROM LFS is easy to undo.

Reply:
Does this read more clearly for you?
```
# Git LFS inclusion / exclusion is controlled by this file. Our approach is to:
# INCLUDE dirs in Git LFS broadly, and then EXCLUDE text file-types narrowly.
#
# Why include in LFS broadly? Once a large binary file is comitted without lfs,
# it is VERY hard to remove it from the git history as it requires a history
# rewrite. This cannot be done without major disruption to all devs. Thus,
# once merged to `staging`, that NNN MB file makes our git repo that much
# bigger forever".
#
# In contrast, if a text file is accidentally included in LFS, adding it back to
# git is easy, see: "HOW TO EXCLUDE A FILE/FILES THATS IN AN LFS DIR" below.
```

I've spent quite a lot of time nuancing this text, I think to some extent it falls under "this is complicated and can't be explained simply". What I've tried to do is not fully explain (i.e. not going to redocument .gitattributes), but provide enough pointers that somebody could figure out what's going on and why if they do some research.

The "sections" I created are there to guide people without full understanding to being able to usefully add/remove things from Git LFS that meet certain patterns without breaking everything.

Reply:
The explanation of include vs exclude was mostly intended to defend against a well-meaning dev 5 years from now deciding to flip the logic (the reverse really is simpler, but I think it makes git history mistakes much more likely and thus raises the chance that somebody 5 years from now will need to do another git history rewrite because .blorgh files became a thing and they're 5GB each and Kelby's replacement started just checking them in)

Reply:
The modification is clear to me! You're right; I was reading it backward. Framing in terms of include/exclude helps.